.github/CODEOWNERS

```diff
@@ -126,3 +126,38 @@ Makefile                    @getsentry/owners-sentry-dev
 /src/sentry/static/sentry/app/utils/discover/                   @getsentry/visibility
 /tests/js/spec/utils/discover/                                  @getsentry/visibility
 ### Endof Perfomance/Discover ###
+
+
+### Ecosystem ###
+
+/src/sentry/api/endpoints/organization_integration_details.py           @getsentry/ecosystem
+/src/sentry/api/endpoints/organization_integration_repos.py             @getsentry/ecosystem
+/src/sentry/api/endpoints/organization_integration_request.py           @getsentry/ecosystem
+/src/sentry/api/endpoints/organization_integrations.py                  @getsentry/ecosystem
+/src/sentry/api/endpoints/organization_plugins_configs.py               @getsentry/ecosystem
+/src/sentry/api/endpoints/organization_plugins.py                       @getsentry/ecosystem
+/src/sentry/api/endpoints/organization_release_previous_commits.py      @getsentry/ecosystem
+/src/sentry/api/endpoints/sentry_internal_app_token_details.py          @getsentry/ecosystem
+/src/sentry/api/endpoints/sentry_internal_app_tokens.py                 @getsentry/ecosystem
+
+/src/sentry/static/sentry/app/views/organizationIntegrations  @getsentry/ecosystem
+
+/src/sentry/integrations/                     @getsentry/ecosystem
+/src/sentry/identity/                         @getsentry/ecosystem
+/src/sentry/plugins/                          @getsentry/ecosystem
+/src/sentry/shared_integrations/              @getsentry/ecosystem
+/src/sentry/mediators/                        @getsentry/ecosystem
+
+
+/src/sentry/models/integration.py             @getsentry/ecosystem
+/src/sentry/models/integrationfeature.py      @getsentry/ecosystem
+
+/src/sentry/tasks/integrations.py             @getsentry/ecosystem
+
+/src/sentry_plugins/                          @getsentry/ecosystem
+
+# To find matching files -> find . -name "*sentry_app*.py"
```

Comment:
Eh, don't think we need this comment. The next two lines are pretty clear. :)

Reply:
I agree, it's not necessary.